api/android/build-android-lib.sh

```diff
@@ -44,6 +44,9 @@
 ##@@   --enable_snpe=(yes|no)
 ##@@       'yes'      : build with sub-plugin for SNPE
 ##@@       'no'       : [default]
+##@@   --enable_pytorch=(yes|no)
```

Comment:
How about using the same way with `tflite_ver`?
For example, 
```(yes(:(1.8.0))?|no)``` and `default is 1.8.0`

Reply:
I updated the code to after the `tflie_ver`. Thanks for suggesting!